### DISCUSSION TITLE
Multiscreen Support

### DISCUSSION THREAD

**Discussion by kadetigor** [cymian/mouseless#64] [Ideas]
It would be great to have a multiscreen support

**Comment by cymian** [comment]
Hi @kadetigor, by default you can move the overlay between monitors with a quick `tap` of `ShiftLeft` or `ShiftRight`.  Please see the [documentation](https://github.com/croian/mouseless-issues/blob/main/mouseless_documentation.md) for more info.

Other ways of making multi-display usage easier will be coming in a future release!

Please let me know if you have any issues, or ideas on how you'd like to see multiple monitor support implemented!
